lib/src/room.dart

```diff
@@ -876,7 +875,7 @@ class Room {
           txid: messageID,
         );
       } catch (e, s) {
-        if (e is MatrixException || DateTime.now().isAfter(timeoutDate)) {
+        if ((e is MatrixException && e.error != MatrixError.M_LIMIT_EXCEEDED) || DateTime.now().isAfter(timeoutDate)) {
```

Comment:
why not delay at exactly time back-end required?

Reply:
Done!
![Screenshot 2023-08-04 at 17 06 03](https://github.com/linagora/matrix-dart-sdk/assets/12546908/59201fb6-a0d5-4731-a859-a4112a4b943b)